ext/spl/spl_dllist.stub.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+class SplQueue extends SplDoublyLinkedList
+{
+    /**
+     * @param mixed $value
+     * @return bool
+     */
+    public function enqueue($value) {}
+
+    /** @return mixed */
+    public function dequeue() {}
+}
+
+class SplDoublyLinkedList implements Iterator, Countable, ArrayAccess, Serializable
+{
+    /**
+     * @param mixed $index
+     * @param mixed $value
+     * @return void
+     */
+    public function add($index, $value) {}
+
+    /** @return mixed */
+    public function pop() {}
+
+    /** @return mixed */
+    public function shift() {}
+
+    /**
+     * @param mixed $value
+     * @return bool
+     */
+    public function push($value) {}
+
+    /**
+     * @param mixed $value
+     * @return void
```

Comment:
```suggestion
     * @return bool
```

Reply:
Ah yes, so these always true return values shouldn't be cleaned up?

Reply:
I think especially in these cases where even the docs say it returns `void`, it's okay to change it. But better as a separate change with a small UPGRADING note.

Reply:
OK, will commit it separately.